### ISSUE TITLE
metadata support?

### ISSUE THREAD

**Issue by johnjack1** [kahing/goofys#233]
How can I set metadata "cache-control" on objects? Does it support "ahbe conf" file like s3fs?

**Comment by kahing** [comment]
nope, I don't have any immediate plan to implement that. For anyone wanting to submit a PR, I think the right approach is to support a virtual xattr and pass-through them as headers.
